autopts/bot/iut_config/zephyr.py

```diff
@@ -345,6 +345,21 @@
             'GAP/SEC/SEM/BI-08-C',
         ]
     },
+
+    "gap_le_sec_mode3_bis.conf": {
+        "overlay": {
+            'CONFIG_BT_ISO_BROADCASTER': 'y',
+            'CONFIG_BT_ISO_SYNC_RECEIVER': 'y',
+            'CONFIG_BT_ISO_CENTRAL': 'y',
+            'CONFIG_BT_ISO_PERIPHERAL': 'y',
+        },
+        "test_cases": [
+            'GAP/SEC/SEM/BV-31-C',
+            'GAP/SEC/SEM/BV-32-C',
+            'GAP/SEC/SEM/BV-34-C',
+            'GAP/SEC/SEM/BV-35-C',
```

Comment:
What about the 

GAP/BIS/BBM/BV-01-C: Broadcast Isochronous Stream Broadcasting mode
GAP/SEC/SEM/BI-13-C: LE security mode 3 – Observer, Reject Lower Level Security
GAP/BIS/BSE/BV-01-C: Broadcast Isochronous Synchronization Establishment procedure

tests?